multisite/management/commands/update_public_suffix_list.py

```diff
@@ -20,17 +20,8 @@ def handle_noargs(self, **options):
         )
         self.log("Updating {filename}".format(filename=filename))
 
-        with tempfile.NamedTemporaryFile(dir=os.path.dirname(filename)) as f:
-            tmpname = f.name
-
-            extract = tldextract.TLDExtract(fetch=True, cache_file=tmpname)
-            extract._get_tld_extractor()
-            self.log(
-                "Downloaded new data to {filename}".format(filename=tmpname)
-            )
-            os.rename(tmpname, filename)
-            f.delete = False    # No need to delete f any more.
-
+        extract = tldextract.TLDExtract(fetch=True, cache_file=filename)
```

Comment:
Oh yeah, it'd be a good idea to have a separate PR for v2 support. It was just that if you removed the kwarg now it'd still work on both so would save on having to change it again later. Up to you!